internal/pkg/term/prompt/prompt.go

```diff
@@ -292,6 +292,13 @@ func WithFinalMessage(msg string) PromptConfig {
 	}
 }
 
+// WithConfirmFinalMessage sets a short final message for to confirm the user's input.
+func WithConfirmFinalMessage() PromptConfig {
+	return func(p *prompt) {
+		p.FinalMessage = color.Emphasize("Sure?")
```

Comment:
What does this look like 🤔 Will it be a breaking change for script users or it is just pure UI changes?

Reply:
It's pure UI changes. So for example, we go from:
```
Are you sure you want to delete application demo? [? for help] (Y/n) 
Are you sure you want to delete application demo?  Y
```
to
```
Are you sure you want to delete application demo? [? for help] (Y/n) 
Sure?  Y
```